### PR TITLE
Bug 1164278. Fx Family nav FxOS distinction.

### PR DESCRIPTION
**Pull request by jpetto**

### CODE REVIEW

**Comment by jpetto** [comment]
May have last minute design changes coming in...

File: bedrock/base/templates/macros.html [hunk 1]
```diff
@@ -583,7 +583,8 @@
             <li><a href="{{ url('firefox.sync') }}" data-id="android-sync"><div>{{ _('Sync') }}</div></a></li>
           </ul>
         </li>
-        <li>
+        <li class="fxos">
+          <span class="separator"></span>
```
Comment (craigcook) [inline bedrock/base/templates/macros.html]: Could this be done with a <code>:before</code> pseudoelement?

Comment (craigcook) [inline bedrock/base/templates/macros.html]: Oh I see, you're using <code>:before</code> and <code>:after</code> on the span itself... maybe this is necessary after all.

**Comment by jpetto** [comment]
The dotted lines are present in the second commit. If this is the direction we like and this is r+'d, someone can squash the commits while I'm out? Otherwise, can wait until/after Whistler.

**Comment by alexgibson** [comment]
Could these dotted separator images be bundled in the image sprite maybe? https://github.com/mozilla/bedrock/blob/master/media/img/firefox/family/nav-sprite5.png

**Comment by jpetto** [comment]
Changes a-comin' based on discussions in Whistler. Should have this updated in a day or two.

**Comment by jpetto** [comment]
Ready for final review. Final change set includes:

- Change "OS" to "Firefox OS" in primary nav
- Remove hover interactions
- Change separator line in sub nav from solid to dotted line

File: bedrock/base/templates/macros.html [hunk 2]
```diff
@@ -558,38 +553,48 @@
...
+            <li><a href="{{ url('firefox.android.index') }}"{% if activesub == 'index' %} class="selected"{% endif %}><div>{{ _('Overview') }}</div></a></li>
+            <li><a href="{{ url('firefox.sync') }}"{% if activesub == 'sync' %} class="selected"{% endif %}><div>{{ _('Sync') }}</div></a></li>
           </ul>
         </li>
-        <li>
-          <a href="{{ url('firefox.os.index') }}" class="primary-link" data-id="os"><div>{{ _('OS') }}</div></a>
+        <li{% if activeprimary == 'os' %} class="active"{% endif %}>
+          <a href="{{ url('firefox.os.index') }}" class="primary-link"{% if activesub == 'index' %} class="selected"{% endif %}>
+            <div>
+            {% if l10n_has_tag('fx_family_nav_update') %}
```
Comment (flodolo) [inline bedrock/base/templates/macros.html]: Let's just put `{{ _('Firefox OS') }}`here without the conditional, I don't plan to expose the string for translation (inside the product the brand "Firefox OS" is not localizable anyway).

**Comment by jpetto** [comment]
Need to update functional test to account for removal of hover state.

**Comment by jpetto** [comment]
Updated tests and removed `l10n_has_tag` conditional.

File: bedrock/base/templates/macros.html [hunk 3]
```diff
@@ -558,39 +553,41 @@
...
-            <li><a href="{{ url('firefox.os.devices') }}" data-id="os-devices"><div>{{ _('Devices') }}</div></a></li>
-            <li><a href="{{ url('firefox.partners.index') }}" data-id="os-partners"><div>{{ _('For partners') }}</div></a></li>
-            <li><a rel="external" href="https://marketplace.firefox.com/" data-id="os-marketplace"><div>{{ _('Marketplace') }}<i class="icon-external"></i></div></a></li>
+            <li><a data-id="os-index" href="{{ url('firefox.os.index') }}"{% if activesub == 'index' %} class="selected"{% endif %}><div>{{ _('Overview') }}</div></a></li>
+            <li><a data-id="os-devices" href="{{ url('firefox.os.devices') }}"{% if activesub == 'devices' %} class="selected"{% endif %}><div>{{ _('Devices') }}</div></a></li>
+            <li><a data-id="os-partners" href="{{ url('firefox.partners.index') }}"{% if activesub == 'partners' %} class="selected"{% endif %}><div>{{ _('For partners') }}</div></a></li>
+            <li><a data-id="os-marketplace" rel="external" href="https://marketplace.firefox.com/" data-id="os-marketplace"><div>{{ _('Marketplace') }}<i class="icon-external"></i></div></a></li>
```
Comment (alexgibson) [inline bedrock/base/templates/macros.html]: Duplicate `data-id` here

**Comment by alexgibson** [comment]
Aside from the one minor nit, this looks good to me! r+wc

I did notice an unrelated bug in old IE, but it seems this is a side effect of the recent change to the aurora CSS gradient, for which I'll file a new bug: 

aurora-css-ie

**Comment by alexgibson** [comment]
Filed [bug 1179196](https://bugzilla.mozilla.org/show_bug.cgi?id=1179196)

**Comment by jpetto** [comment]
Fixed the duplicate `data-id`. Thanks @alexgibson!

**Comment by alexgibson** [comment]
r+ :rabbit:

**Pull request merged**